src/seedu/addressbook/data/member/Member.java

```diff
@@ -11,7 +11,7 @@
  */
 public class Member implements ReadOnlyMember {
 
-    public static final String EMPTY_NAME_STRING = "EMPTY";
+    public static final String EMPTY_NAME_STRING = "2l65D1f5H";
```

Comment:
Is this supposed to be edited?

Reply:
I expected someone to come up with the "EMPTY" name String during testing.
So I use a random string generator to make it near impossible to accidentally break the program.